packages/react-reconciler/src/__tests__/ReactExpiration-test.js

```diff
@@ -733,13 +733,19 @@ describe('ReactExpiration', () => {
       // Both normal pri updates should have expired.
       expect(Scheduler).toFlushExpired([
         'Sibling',
-        // Note: we also flushed the high pri update here, because in the
-        // current implementation, once we pick the next lanes to work on, we
-        // entangle it with all pending at equal or higher priority. We could
-        // feasibly change this heuristic so that the high pri update doesn't
-        // render until after the expired updates have finished. But the
-        // important thing in this test is that the normal updates expired.
```

Comment:
I love when a test fails and there's a comment that anticipates the exact reason. Thank you to the very smart, cool, and interesting person who wrote it! (me if that wasn't clear)